delta/migration/scripts/cassandra-init/init.sh

```diff
@@ -0,0 +1,5 @@
+until cqlsh -f /cassandra-init/cassandra_1_5.ddl && cqlsh -f /cassandra-init/create-materialized-view.ddl && sstableloader --nodes localhost  /cassandra-backups/cassandra-0/delta_1_5/messages && sstableloader --nodes localhost  /cassandra-backups/cassandra-1/delta_1_5/messages && sstableloader --nodes localhost  /cassandra-backups/cassandra-2/delta_1_5/messages; do
```

Comment:
cassandra-0, cassandra-1 and cassandra-2 have the same messages, no need to load them all

Reply:
Fixed